### ISSUE TITLE
Menu link text too dark when active and in dark mode

### ISSUE THREAD

**Issue by ReinhardtBotha** [reorx/hugo-PaperModX#26]
The menu link text color is dark when active and the theme is in dark mode, which makes it illegible.  Here are screenshots with the Tags link active.

Dark Mode: 
![image](https://user-images.githubusercontent.com/89278135/189502338-e5fb0b66-f341-49cf-90b2-ffb72da59511.png)

Light Mode:
![image](https://user-images.githubusercontent.com/89278135/189502299-fd22e08c-7c5f-41fc-8345-ddb1a1bc086d.png)

**Comment by BedirT** [comment]
I don't think that is about the theme. I believe you are using some additional CSS to modify the navigation bar.

**Comment by ReinhardtBotha** [comment]
I also thought so, but it does the same on the demo site (https://reorx.github.io/hugo-PaperModX/). Maybe it's due to the latest CSS updates to the theme?

**Comment by BedirT** [comment]
Just found out what causes this. After updating to the most recent version I got the same error. It is due to these lines here:

https://github.com/reorx/hugo-PaperModX/blob/d7986e71bfeda9ca660e0253b7d6180093e23c9d/assets/css/core/theme-vars.css#L30-L32

He is using the `signature` as the colour variable which does not change based on the theme dark/light version. It needs a fix!

**Comment by itsamemarkus** [comment]
Thanks for the pull request! Noticed that too.

Just wondering if there is a bigger problem with light/dark theme because [accent colors](https://github.com/reorx/hugo-PaperModX#give-links-an-accent-color) defined in `assets/css/extended/custom.css` only work for the light theme, right?
